File: result/translate/chatgpt/java2cpp/c++/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.cpp
int f_gold(int arr[], int n, int k) {
  priority_queue<int, vector<int>, greater<int>> pq;
  for (int i = 0; i < n; i++) pq.push(arr[i]);
  int count = 0, ans = 1;
  while (!pq.empty() && count < k) {
    ans += pq.top();
    pq.pop();
    count++;
  }
  return ans;
}

File: result/translate/chatgpt/python2java/java/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.java
public int f_gold(int[] arr, int n, int k) {
    Queue<Integer> pq = new PriorityQueue<>();
    for (int i = 0; i < n; i++) {
        pq.add(arr[i]);
    }
    int count = 0;
    int ans = 1;
    while (!pq.isEmpty() && count < k) {
        ans += pq.poll();
        count++;
    }
    return ans;
}
